models/bridge/readme.md

```diff
@@ -0,0 +1,20 @@
+# Bridges Spells
```

Comment:
absolutely love this

we should make it into a doc block and add it to the model description (we can do that after merging tho)

Reply:
Yeah, open to see what that looks like after (I'm still a dbt noob)